octobot_trading/enums.py

```diff
@@ -381,9 +392,22 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     REDUCE_ONLY = "reduceOnly"
     STOP_PRICE = "stopPrice"
     TRIGGER_ABOVE = "triggerAbove"
+    POST_ONLY = "post_only"
```

Comment:
Are we supporting it now?

Reply:
Is it supported by OrderParser?

Reply:
Yeah the parser implementation is done. Only the Order class need to be change so its using the parsed value.
Depending on the exchange, it might require custom logic to work properly. I think binance should work out of the box